_pages/release-strategies.md

```diff
@@ -0,0 +1,115 @@
+issues. 
+
+#### Release to Different User Groups at Different Times
+
+Recommendation: Releasing to a subset of users at a time, creating a better user experience for them and a better time post-release for
+your team. 
+
+There are always going to be immediate bug fixes and customer asks in the aftermath of a release. 
+Initally scoping a big release to a subset of user types at a time will narrow the developer and customer success team's focus, making
+debugging and prioritizing fixes easier.
+
+#### Practice Data Migrations Often
+
+Recommendation: Do dry runs of any critical data or infrastructure migrations.
+
+Practice makes perfect! Try out your migrations regularly in the months leading up to the release so that everyone feels very comfortable 
+with how they work and what to expect. 
+
+#### Develop Mature Incident Response Practices
+
+Recommendation: Build [Incident Response practices and run drills](/security/incident-response-drills/) before releasing.
+
+It's impossible to completely de-risk a release. Developing a plan ahead of time addressing how you will approach fan incident will
+enable your team to focus on fixing the issue as soon as possible. Once you have a response plan, you should conduct 
+[incident type drills](/security/incident-response-drills/#example-incident-response-drills) so that your team is well-practiced in
+what do if something goes wrong.
+
+#### Develop Training Strategies in Advance
+
+Recommendation: Do user research and develop training strategies ahead of time to help users ease into the new system.  
+
+It can be jarring as a user to one day have a completely new system. Spending some time before release working with critical users of the
+system to understand the common pitfalls a user may experience or uncover will help you develop training materials to address those issues 
+(or make the system more intuitive.) Releasing training materials for users to look at before the release will create better familiarity and
+make users less wary or uncomfortable with the big change.
+
+#### Prepare Customer Support Response Templates
+
+Recommendation: Develop templates for expected support requests to have consistent messaging and advice to users.
+
+Launches often have expected support requests, such as: 
+* "Why doesn't the system do X?"
+* "This is broken."
+* "I can't find Z."
+
+Preparing responses ahead of time allows the customer support team to give the same suppport to anyone who a predictable issue, allowing them
+to focus their energies on unexpected support needs. It also helps unsure a better customer experience for users.
+
+### Big Bang Release Factors
+
+There are some factors that make partners more attached to big bang releases. There are some strategies you can deploy to try to 
+convince partners to pivot to a smaller, more iterative release strategy.
```

Comment:
I would suggest the number one thing you can do  -- if you can manage it -- is getting the partner to pivot from a project-centric to a product-centric approach, which pays off in about 100 different ways. In the former, the partner sees the big bang release as the culmination and end point of the project, which is *really* problematic and one of the biggest issues gov IT faces. In the latter, they understand that the release is just one milestone in the lifecycle. I know we've written a lot about that project-to-product shift in the past (and I'm sure the product chapter would happily weigh in). At least IMO, that's the biggest and important nut to crack.

Reply:
I think we've almost definitely written about it many times before, so maybe we can reference and link?

Reply:
i did a quick search of the 18f blog and couldn't find it specifically; do you have a link you could pass along?

Reply:
I wonder if this series would work well to link to? https://18f.gsa.gov/2019/08/22/building-product-management-capacity-in-government-part-1/